Gemfile

```diff
@@ -12,7 +12,7 @@ platforms :ruby do
   gem 'sqlite3'
 end
 
-platforms :mri_21 do
+platforms :mri_21, :mri_22 do
```

Comment:
> Only supports MRI 2.0.0 or newer. For MRI 1.9.3 or older, use pry-debugger

This is only a tool for development, so I think that would be fine. Specially since sidekiq doesn't support 1.9. I'll change it!